docs/reference/configuration.txt

```diff
@@ -395,8 +395,8 @@ for details on driver options.
       # When chaining the same operators that use the same field, setting this
       # feature flag to false will cause those operators to be combined using an
       # and. Setting this feature flag to true will cause the later chained
-      # operators to overwrite the earlier ones. (default: false)
-      #overwrite_chained_operators: false
+      # operators to overwrite the earlier ones. (default: true)
```

Comment:
Please follow the comment format used elsewhere in this file, if this is indeed the case:

```
      # (default: true in Mongoid 7, will change to false in Mongoid 8)
```